src/organizations/components/OrgProfileTab/index.tsx

```diff
@@ -149,11 +156,18 @@ const OrgProfileTab: FC = () => {
       </FlexBox>
 
       {CLOUD && orgDetailsLoaded && (
-        <FlexBox.Child className="org-profile-tab--section">
+        <FlexBox
+          direction={FlexDirection.Row}
+          stretchToFitWidth={true}
+          className="org-profile-tab--section"
+        >
           <UsersProvider>
-            <DeletePanel />
+            <>
+              <DeletePanel />
+              {allowSelfRemoval && showLeaveOrgButton && <LeaveOrgButton />}
+            </>
```

Comment:
I think I would put the "Leave" org button above delete, rather than putting it below Delete, since "Deleting" the org is a lot more final than leaving it.

Reply:
good call